app/components/common/highLightedContent/index.tsx

```diff
@@ -49,7 +50,7 @@ export const STOP_WORDS = [
   "will",
   "with",
 ];
-const MAX_LENGTH_OF_ABSTRACT_EXTEND = 400;
+const MAX_LENGTH_OF_ABSTRACT_EXTEND = 1050;
```

Comment:
It's duplicated code. please import it from `app/components/common/paperItem/abstract.tsx`

Reply:
and I think of that if we use it as a `component`, we should get this value from `props`.
example
```
interface HighLightContentProps {
  ...
  maxCharLimit: number;
}
```